docs/software_guide.md

```diff
@@ -28,11 +28,11 @@ To add a new test suite, add a submodule containing the suite to bp_common/test/
 ## Building a Checkpoint Test
 BlackParrot can use Dromajo to generate checkpoints for certain tests. It runs the test on Dromajo for a certain number of instructions and then generates a memory image and a bootrom image which loads the internal architectural state of the cores(PC, registers, CSRs, privilege mode, ...). We use the `sim_sample` target to create a checkpoint at `SAMPLE_START_P=<n>` for RTL simulation. We have to run it with `NBF_CONFIG_P=1` to load the processor configuration using the nbf loader and also use a bootrom configuration to load the internal state of the cores.
 
-Optionally we can use `PRELOAD_MEM_P=1` to preload the memory image instead of writing it using the nbf loader and `SAMPLE_MEMSIZE=<k in MB>` to specify the size of memory image(default is 128MB).
+Optionally we can use `SAMPLE_MEMSIZE=<k in MB>` to specify the size of memory image(default is 128MB).
```

Comment:
It seems like memory is always loaded by NBF loader now? (excepting solution for Farzam's question regarding Linux parallel cosim)

Reply:
Oh yeah let me revert this, originally I thought we had to get rid of the preloading